### PR TITLE
LPD-15362 Trim namespace in PublicationTimeline

### PR DESCRIPTION
**Pull request by ctang3**
While verifying https://liferay.atlassian.net/browse/LPD-15362 after it was merged into master, I noticed that the action links for each timeline item were broken. 

The namespace PublicationTimeline.js gets from it's parent is formed by this call in the backend:  `_portal.getPortletNamespace(CTPortletKeys.PUBLICATIONS)`, where `getPortletNamespace(...)` wraps the namespace in underscores for it to be used in certain other requests.

### CODE REVIEW

**Comment by ctang3** [comment]
ci:test:relevant

**Comment by ctang3** [comment]
ci:test:publications

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js [hunk 1]
```diff
@@ -31,7 +31,10 @@ const PublicationTimeline = ({namespace, timelineItemsURL}) => {
 			{
 				ctCollectionId,
 				mvcRenderCommandName,
-				p_p_id: namespace,
+				p_p_id:
+					namespace.startsWith('_') && namespace.endsWith('_')
+						? namespace.slice(1, namespace.length - 1)
+						: namespace,
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js:37]: There's a util to get the portletId from the portletNamespace: [getPortletId](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/get_portlet_id.js).

You can use `Liferay.Util.getPortletId` or import it from `frontend-js-web`.

**Comment by ctang3** [comment]
@pei-jung updated, thanks!

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**